spec/models/task_spec.rb

```diff
@@ -32,4 +32,18 @@
         expect(task.is_completed).to be false
     end
 
-end
\ No newline at end of file
+    it "will update task.is_completed from true to false" do
+        user = User.create(first_name: "FirstName", 
+            last_name: "LastName", 
+            email: "Email",
+            username: "Username",
+            password: "Password",
+            password_confirmation: "Password")
+        task1 = Task.create(description: "walk the dog", user_id: user.id)
+        task2 = Task.create(description: "feed the dog", user_id: user.id)
+        task1.update(is_completed: true)
+        Task.update_task
+
+        expect(Task.all[0].is_completed).to be false
+    end
```

Comment:
I'm a bit confused as to what this test is doing. It seems like it's only testing one task, but there are two tasks being created 🤔

Reply:
yes, I don't need the second task